docs/Changelog.md

```diff
@@ -14,6 +14,7 @@
 * [#5780](https://github.com/TouK/nussknacker/pull/5780) Fixed Scala case classes serialization when a class has additional fields in its body
 * [#5853](https://github.com/TouK/nussknacker/pull/5853) Add processing mode information and filtering to the components table
 * [#5843](https://github.com/TouK/nussknacker/pull/5843) Added new endpoint to provide statistics URL
+* [#5887](https://github.com/TouK/nussknacker/pull/5887) Fix toolbar icon styles when multiline text
```

Comment:
Can you add info about jira issue in this line?

Reply:
good question, I reckon that we had this discussion internally in the team, and the conclusion was, that since our Jira is private, there is no use in adding it since users won't have access to it.  Isn't the Jira task number in a PR enough?